include/mbedtls/check_config.h

```diff
@@ -708,48 +708,9 @@
 #if defined(MBEDTLS_SHA512_ALT) || defined(MBEDTLS_SHA512_PROCESS_ALT)
 #error "MBEDTLS_SHA512_*ALT can't be used with MBEDTLS_SHA512_USE_A64_CRYPTO_*"
 #endif
-/*
- * Best performance comes from most recent compilers, with intrinsics and -O3.
```

Comment:
We should retain more of this comment as there's some useful info which is lost in this PR

Reply:
Yes. I will add it back. 
But I am still thinking about how to add it back in command line. clang can only work with command line now.

Reply:
reverted.